### PR TITLE
Adding the "getting started" tutorial content

### PR DESCRIPTION
**Pull request by spbail**
Added the contents of the GE getting started tutorial to this. I think the tutorial has 2 use cases:
1) tutorial - in which case we'd expect users to just use the Docker container for the postgres database and go through the tutorial by creating a new project directory from scratch
2) demo/explore GE - I think for this use case we might want to bundle this with the GE image, so it doesn't depend on a local installation of GE and dependencies (SQLalchemy, psycopg2 etc)

Let me know what you think!

### CODE REVIEW

File: ge_dbt_airflow_tutorial/Dockerfile [hunk 1]
```diff
@@ -72,9 +72,9 @@ RUN set -ex \
         /usr/share/doc \
         /usr/share/doc-base
 
-COPY deploy/script/entrypoint.sh /entrypoint.sh
+COPY deploy /entrypoint.sh
```
Comment (spbail) [inline ge_dbt_airflow_tutorial/Dockerfile:75]: This change doesn't look right and I don't think I made it, I'll roll back.

**Comment by spbail** [comment]
Also FYI @abegong, just for visibility.

File: README.md [hunk 2]
```diff
@@ -1,109 +1,11 @@
-# Great Expectations Pipeline Tutorial
+# Great Expectations tutorials
```
Comment (spbail) [inline README.md:1]: Most of the content here was moved into the README in the ge_dbt_airflow_tutorial subdirectory.

File: ge_dbt_airflow_tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+# Great Expectations dbt + Airflow Pipeline Tutorial
```
Comment (spbail) [inline ge_dbt_airflow_tutorial/README.md:1]: This was just moved directly from the README in the root directory, adjusted a couple of references to paths.

File: ge_getting_started_tutorial/great_expectations/checkpoints/taxi.demo.staging.chk.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# This checkpoint was created by the command `great_expectations checkpoint new`.
```
Comment (spbail) [inline ge_getting_started_tutorial/great_expectations/checkpoints/taxi.demo.staging.chk.yml:1]: This checkpoint and the expectation suite are just direct copies of the versions in ge_demo, so have been "reviewed"/used. Feel free to just skim.

File: ge_getting_started_tutorial/great_expectations/notebooks/pandas/validation_playground.ipynb [hunk 5]
```diff
@@ -0,0 +1,240 @@
+{
```
Comment (spbail) [inline ge_getting_started_tutorial/great_expectations/notebooks/pandas/validation_playground.ipynb:1]: These are all just default GE setup, feel free to ignore.

File: ge_getting_started_tutorial/slides/webinar_ge_101.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
+---
```
Comment (spbail) [inline ge_getting_started_tutorial/slides/webinar_ge_101.md:1]: These are also copied over from the ge_demo repo, so no need to review unless you spot something obviously wrong.

**Review by spbail — COMMENTED** [review]
FYI @jcampbell @abegong @eugmandel I added some comments on how to review this PR. The majority of changes is files that were moved from the root into the ge_dbt_airflow_tutorial subdirectory. Any other changes you should be able to just review top to bottom. Thank you!

**Review by eugmandel — APPROVED** [review]

**Pull request merged**